.travis.yml

```diff
@@ -2,6 +2,10 @@ sudo: required
 
 dist: trusty
 
+branches:
+  only:
+  - master
```

Comment:
Wait doesn't this end up stopping pull request builds?

Reply:
Its funny I suspected the same thing initially but then I checked that this is a separate repo setting called `Build pushed Pull Request` here : https://travis-ci.org/fission/fission/settings
https://docs.travis-ci.com/user/pull-requests/ for reference and this PR did run a CI test.